templateUtils.tsx

```diff
@@ -1200,6 +1202,26 @@ export const getSpacingBetweenSections = (
     return spacingAccortingToType?.[spacing_between_sections];
 };
 
+export const getBodyHtmlMCERT = (heading: string, description: string) => {
```

Comment:
Let's move it out to a new file? templateUtils is already too overwhelmed. What do you say?